### PR TITLE
Release changes 1.15

### PR DESCRIPTION
**Pull request by TysonStanley**
The changes made during the release process to CRAN (minor adjustments and version bump for the tag).

### CODE REVIEW

File: src/init.c [hunk 1]
```diff
@@ -353,6 +353,6 @@ SEXP initLastUpdated(SEXP var) {
 
 SEXP dllVersion(void) {
   // .onLoad calls this and checks the same as packageVersion() to ensure no R/C version mismatch, #3056
-  return(ScalarString(mkChar("1.14.99")));
+  return(ScalarString(mkChar("1.15.0")));
```
Comment (tdhock) [inline src/init.c:356]: I thought we did not want 1.15.0 (or any even number release version) to get on master branch?

**Review by tdhock — CHANGES_REQUESTED** [review]

File: src/init.c (hunk 1 above)
Comment (TysonStanley) [reply src/init.c:356]: For the tag, it must be changed to match the tag itself. This matches the release to CRAN so should not produce any issues. However, not sure if this should be merged to master (rather the 1.15.99).

Comment (tdhock) [reply src/init.c:356]: yes I think it is normal to have the version number match the tag name, but I thought we kept that code on a separate branch? (not master) 
correct me if I'm wrong.

Comment (TysonStanley) [reply src/init.c:356]: I think that is what we did previously, so yes, let's not include this last commit in the PR. I'll remove those changes from this PR.

**Comment by TysonStanley** [comment]
Added version bump to 1.15.99 for merging to master

**Comment by ben-schwen** [comment]
@TysonStanley could you also update the NEWS file to 1.15.0, stating when it was submitted to CRAN and then retag 1.15.0 (at least that's what Matt always did)

**Comment by TysonStanley** [comment]
Using this branch for the tag. Would recommend merging at least the first two commits here for the master branch when we bring 1.15.99 into master.

**Comment by MichaelChirico** [comment]
> Using this branch for the tag. Would recommend merging at least the first two commits here for the master branch when we bring 1.15.99 into master.

planning to rebase the 1-15-99 branch once this is in master to have the cleanest possible history

**Comment by TysonStanley** [comment]
Sounds good. The tag is created. Do you want me to create a new PR with just the relevant commits for master?

**Comment by ben-schwen** [comment]
I guess we are good with this PR

Closes #5931 #5924

**Comment by tdhock** [comment]
Hi, I don't know if we still care to follow CRAN_Release.cmd, which says the following:
```
# DO NOT commit or push to GitHub. Leave 4 files (.dev/CRAN_Release.cmd, DESCRIPTION, NEWS and init.c) edited and not committed. Include these in a single and final bump commit below.
# DO NOT even use a PR. Because PRs build binaries and we don't want any binary versions of even release numbers available from anywhere other than CRAN.
# Leave milestone open with a 'release checks' issue open. Keep updating status there.
# ** If on EC2, shutdown instance. Otherwise get charged for potentially many days/weeks idle time with no alerts **
# If it's evening, SLEEP.
# It can take a few days for CRAN's checks to run. If any issues arise, backport locally. Resubmit the same even version to CRAN.
# CRAN's first check is automatic and usually received within an hour. WAIT FOR THAT EMAIL.
# When CRAN's email contains "Pretest results OK pending a manual inspection" (or similar), or if not and it is known why not and ok, then bump dev.

###### Bump dev for NON-PATCH RELEASE
# 0. Close milestone to prevent new issues being tagged with it. The final 'release checks' issue can be left open in a closed milestone.
# 1. Check that 'git status' shows 4 files in modified and uncommitted state: DESCRIPTION, NEWS.md, init.c and this .dev/CRAN_Release.cmd
# 2. Bump minor version in DESCRIPTION to next odd number. Note that DESCRIPTION was in edited and uncommitted state so even number never appears in git.
```
The last line is what I'm worried about (if we still care to follow these instructions). The "even number never appears in git" means that we should not push any commits with version 1.15.0 to master branch, and instead master should go from 1.14.99 to 1.15.99 -- does that make sense? Also it says below that the specific commit message we should use:
```r
# 8. Push to master with this consistent commit message: "1.14.8 on CRAN. Bump to 1.14.10"
```
so if you agree, let's close this PR and instead merge #5936 ?

**Comment by jangorecki** [comment]
The other PR looks better but there are good updates to release procedure here in this PR that make sense to include as well, but probably better not in version bump PR anyway

**Pull request closed without merging**